playground/infrastructure/cd_helper.py

```diff
@@ -44,8 +46,13 @@ def store_examples(self, examples: List[Example]):
     """
     Store beam examples and their output in the Google Cloud.
     """
+    logger.info("Start of executing Playground examples ...")
     asyncio.run(self._get_outputs(examples))
+    logger.info("Start of executing Playground examples ...")
```

Comment:
It is the same as above on line 49, is it correct?

Reply:
Changed.